src/realizations/catchment/Tshirt_C_Realization.cpp

```diff
@@ -99,15 +100,14 @@ Tshirt_C_Realization::Tshirt_C_Realization(forcing_params forcing_config,
                                                                               Cgw, expon, max_gw_storage),
                                                         nash_storage)
 {
-    _link_legacy_forcing();
 }
 
 Tshirt_C_Realization::Tshirt_C_Realization(
         std::string id,
         forcing_params forcing_config,
         utils::StreamHandler output_stream
-) : Catchment_Formulation(std::move(id), std::move(std::make_unique<Forcing>(forcing_config)), output_stream) {
-    _link_legacy_forcing();
+//) : Catchment_Formulation(std::move(id), std::move(std::make_unique<Forcing>(forcing_config)), output_stream) {
+) : Catchment_Formulation(std::move(id), std::move(std::make_unique<CsvPerFeatureForcingProvider>(forcing_config)), output_stream) {
```

Comment:
Same comment as above... where is this constructor used, and does this need to be non-specific to CsvPerFeature?

Reply:
I would tend to agree...